assets/templates/partials/json-ld/dataset/common.tmpl

```diff
@@ -1,5 +1,20 @@
+{{$PageData := .}}
+"name": {{ .Metadata.Title }},
+"publisher": {
+    "@type": "GovernmentOrganization",
```

Comment:
I know this was actually here from before, but Google doesn't actually accept the specific sub type `GovernmentOrganization` for the publisher, so this should be changed to `Organization`